keyboards/nix_studio/n60a/n60a.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Nix Studio
```

Comment:
License type needs to be specified. 
```suggestion
// Copyright 2022 Nix Studio
// SPDX-License-Identifier: GPL-2.0-or-later
```